mopidy/core/playback.py

```diff
@@ -301,7 +301,8 @@ def play(self, tl_track=None, tlid=None):
 
         if tl_track is not None:
             # TODO: allow from outside tracklist, would make sense given refs?
-            assert tl_track in self.core.tracklist.get_tl_tracks()
+            if (tl_track not in self.core.tracklist.get_tl_tracks()):
```

Comment:
Parenthesis are not needed here:

```suggestion
            if tl_track not in self.core.tracklist.get_tl_tracks():
```

Reply:
This was also caught by the Black formatting check at CircleCI: https://circleci.com/gh/mopidy/mopidy/1953